src/scenes/Office/MoveInfo.jsx

```diff
@@ -387,20 +389,23 @@ class MoveInfo extends Component {
                 </Alert>
               )}
               <div>
-                {moveInfoComboButton && (
-                  <ComboButton
-                    items={[
-                      { value: 'Approve Basics', disabled: true },
-                      { value: 'Approve HHG', disabled: true },
-                      { value: 'Approve PPM', disabled: true },
-                    ]}
-                    buttonText={'Approve'}
-                    disabled={!ordersComplete}
-                    toolTipText={
-                      'Some information about the move is missing or contains errors. Please fix these problems before approving.'
-                    }
-                  />
-                )}
+                <ToolTip
+                  disabled={ordersComplete}
+                  textStyle={'tooltiptext-large'}
```

Comment:
(nit) String literals don't need to be wrapped in curly braces.

```suggestion
                  textStyle="tooltiptext-large"
```

Reply:
no worries. good to know. nit away!